doc/tutorials/klondike/app/lib/step4/klondike_game.dart

```diff
@@ -45,19 +45,15 @@ class KlondikeGame extends FlameGame {
       ),
     );
 
-    final world = World()
-      ..add(stock)
-      ..add(waste)
-      ..addAll(foundations)
-      ..addAll(piles);
-    add(world);
+    world.add(stock);
+    world.add(waste);
+    world.addAll(foundations);
+    world.addAll(piles);
 
-    final camera = CameraComponent(world: world)
-      ..viewfinder.visibleGameSize =
-          Vector2(cardWidth * 7 + cardGap * 8, 4 * cardHeight + 3 * cardGap)
-      ..viewfinder.position = Vector2(cardWidth * 3.5 + cardGap * 4, 0)
-      ..viewfinder.anchor = Anchor.topCenter;
-    add(camera);
+    camera.viewfinder.visibleGameSize =
```

Comment:
I'm still curious about why the elimination of the cascade operator. It shouldn't change anything here.

Reply:
As before, `camera` is now an built-in object in FlameGame and is inherited. And the Tutorial text is discussing how to define a game's co-ordinate system to suit the objects in the game (i.e. cards)i and why we anchor the camera at top center. So I think more explicit syntax helps to emphasise the key points, which are the parameters used to lay out the game and set up the camera.